detections/application/splunk_protocol_impersonation_weak_encryption_simplerequest.yml

```diff
@@ -0,0 +1,52 @@
+name: Splunk protocol impersonation weak encryption simplerequest
+id: 839d12a6-b119-4d44-ac4f-13eed95412c8
+version: 1
+date: '2022-05-24'
+author: Rod Soto, Splunk
+type: Hunting
+datamodel: []
+search: '`splunk_python` "simpleRequest SSL certificate validation is enabled without hostname verification" | stats count by host path | `splunk_protocol_impersonation_weak_encryption_simplerequest_filter`'
+description: On June 14th 2022, Splunk released vulnerability advisory addresing Python TLS validation which was not set before Splunk version 9. From version 9 on Python3 client libraries now verify server certificates by default and use CA certificate store.This search warns about failure to validate certification using python3 request.
```

Comment:
rename "This search warns about failure to validate certification using python3 request." 

to 

"This search warns a user about a failure to validate a certificate using python3 request."

Reply:
done